specification/containerinstance/resource-manager/Microsoft.ContainerInstance/2017-08-01-preview/containerInstance.json

```diff
@@ -181,13 +185,13 @@
         ],
         "responses": {
           "200": {
-            "description": "OK - the container group was previously created and is now updated",
+            "description": "OK - the container group already exists and is now updated.",
```

Comment:
if there is no spec change, we also return 200, so 200 may be a no-op. Not sure if this information need to be exposed to customer.

Reply:
Good point - removing additional messages from 200 responses to be only "OK".